trashbinFilesFolders.feature:136 is sometimes passing

https://cloud.drone.io/owncloud/ocis/1367/5/7
```
runsh: Total unexpected passed scenarios throughout the test run:
apiTrashbin/trashbinFilesFolders.feature:136
Makefile:207: recipe for target 'test-acceptance-api' failed
make: *** [test-acceptance-api] Error 1
```
and I think this happened once yesterday.

So this test scenario can sometimes pass. Maybe there is a timing issue. It needs investigation.

--- a/tests/acceptance/features/apiTrashbin/trashbinFilesFolders.feature
+++ b/tests/acceptance/features/apiTrashbin/trashbinFilesFolders.feature
@@ -104,7 +104,7 @@ Feature: files and folders exist in the trashbin after being deleted
       | old      |
       | new      |
 
-  @issue-23151
+  @skipOnOcV10 @issue-23151
   # This scenario deletes many files as close together in time as the test can run.
   # On a very slow system, the file deletes might all happen in different seconds.
   # But on "reasonable" systems, some of the files will be deleted in the same second,
@@ -126,10 +126,42 @@ Feature: files and folders exist in the trashbin after being deleted
       | /folderC/textfile0.txt |
       | /folderD/textfile0.txt |
     # When issue-23151 is fixed, uncomment these lines. They should pass reliably.
-    #Then as "Alice" the folder with original path "/folderA/textfile0.txt" should exist in the trashbin
-    #And as "Alice" the folder with original path "/folderB/textfile0.txt" should exist in the trashbin
-    #And as "Alice" the folder with original path "/folderC/textfile0.txt" should exist in the trashbin
-    #And as "Alice" the folder with original path "/folderD/textfile0.txt" should exist in the trashbin
+    Then as "Alice" the folder with original path "/folderA/textfile0.txt" should exist in the trashbin
+    And as "Alice" the folder with original path "/folderB/textfile0.txt" should exist in the trashbin
+    And as "Alice" the folder with original path "/folderC/textfile0.txt" should exist in the trashbin
+    And as "Alice" the folder with original path "/folderD/textfile0.txt" should exist in the trashbin
+    And as "Alice" the folder with original path "/textfile0.txt" should exist in the trashbin
+    Examples:
+      | dav-path |
+      | old      |
+      | new      |
+
+  @notToImplementOnOcis @issue-23151
+  # This scenario deletes many files as close together in time as the test can run.
+  # On a very slow system, the file deletes might all happen in different seconds.
+  # But on "reasonable" systems, some of the files will be deleted in the same second,
+  # thus testing the required behavior.
+  Scenario Outline: trashbin can store two files with the same name but different origins when the files are deleted close together in time
+    Given using <dav-path> DAV path
+    And user "Alice" has created folder "/folderA"
+    And user "Alice" has created folder "/folderB"
+    And user "Alice" has created folder "/folderC"
+    And user "Alice" has created folder "/folderD"
+    And user "Alice" has copied file "/textfile0.txt" to "/folderA/textfile0.txt"
+    And user "Alice" has copied file "/textfile0.txt" to "/folderB/textfile0.txt"
+    And user "Alice" has copied file "/textfile0.txt" to "/folderC/textfile0.txt"
+    And user "Alice" has copied file "/textfile0.txt" to "/folderD/textfile0.txt"
+    When user "Alice" deletes these files without delays using the WebDAV API
+      | /textfile0.txt         |
+      | /folderA/textfile0.txt |
+      | /folderB/textfile0.txt |
+      | /folderC/textfile0.txt |
+      | /folderD/textfile0.txt |
+    # When issue-23151 is fixed, remove this scenario and use the above one.
+#    Then as "Alice" the folder with original path "/folderA/textfile0.txt" should exist in the trashbin
+#    And as "Alice" the folder with original path "/folderB/textfile0.txt" should exist in the trashbin
+#    And as "Alice" the folder with original path "/folderC/textfile0.txt" should exist in the trashbin
+#    And as "Alice" the folder with original path "/folderD/textfile0.txt" should exist in the trashbin
     And as "Alice" the folder with original path "/textfile0.txt" should exist in the trashbin
     Examples:
       | dav-path |

